spec/features/blog_spec.rb

```diff
@@ -29,7 +29,7 @@
       click_button 'Create Article'
 
       expect(current_path).to eq("/articles/#{Article.last.id}")
```

Comment:
Should need to test these paths, as Rails does this for you and is pretty heavily tested within Rails